code/datums/keybinding/living.dm

```diff
@@ -21,7 +21,7 @@
 	return TRUE
 
 /datum/keybinding/living/look_up
-	hotkey_keys = list("L")
+	hotkey_keys = list("") //MONKESTATION EDIT. replace L with nothing
```

Comment:
perhaps use U as the LOOC hotkey, similar to beecode (and i guess monke 1.0, if it wasn't changed then?)

Reply:
I was originally going to just leave it unbound since look up used L, but borbop told me I might as well just unbind look up and replace it with looc.